src/tools/tidy/src/error_codes_check.rs

```diff
@@ -15,21 +15,58 @@ const WHITELIST: &[&str] = &[
     "E0727", "E0729",
 ];
 
+// Some error codes don't have any tests apparently...
+const IGNORE_EXPLANATION_CHECK: &[&str] = &[
+    "E0570", "E0601", "E0602", "E0639", "E0729", "E0749", "E0750", "E0751",
```

Comment:
Open an issue with a list of these error codes and the goal to remove this constant